frontend/public/components/start-guide.tsx

```diff
@@ -5,68 +5,88 @@ import { Link } from 'react-router-dom';
 import { Button } from '@patternfly/react-core';
 
 import { createProjectMessageStateToProps } from '../reducers/ui';
-import { Disabled, HintBlock } from './utils';
+import { Disabled, HintBlock, ExternalLink, openshiftHelpBase, LinkifyExternal } from './utils';
 import { connectToFlags } from '../reducers/features';
 import { FLAGS } from '../const';
 import { ProjectModel, RoleModel, StorageClassModel } from '../models';
+import { createProjectModal } from './modals/create-namespace-modal';
 
 const WHITELIST = [RoleModel.kind, StorageClassModel.kind];
 
-export const StartGuide: React.FC<StartGuideProps> = ({ startGuide }) => {
-  return <div className="co-m-pane__body">{startGuide}</div>;
-};
-
 export const OpenShiftGettingStarted = connect(createProjectMessageStateToProps)(
-  ({ createProjectMessage }) => (
-    <HintBlock title="Getting Started">
-      {createProjectMessage ? (
-        <p className="co-pre-line">{createProjectMessage}</p>
-      ) : (
+  ({ canCreateProject = true, createProjectMessage }: OpenShiftGettingStartedProps) => (
+    <>
+      {canCreateProject ? (
         <p>
           OpenShift helps you quickly develop, host, and scale applications. To get started, create
           a project for your application.
         </p>
+      ) : (
+        <p>
+          OpenShift helps you quickly develop, host, and scale applications. To get started, you'll
+          need a project. Currently, you can't create or access any projects. You'll need to contact
+          a cluster administrator for help.
```

Comment:
I don't think we should show "You'll need to contact a cluster administrator" if there's a custom message.